content/en/kb/repo-directory-does-not-exist/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "StorageMiner repo directory does not exist"
+description: "This is a solution for the StorageMiner repo directory does not exist error."
+date: 2022-03-22T12:00:35+01:00
+lastmod: 2022-03-22T12:00:35+01:00
+draft: false
+menu:
+  kb:
+    parent: "browse"
+toc: false
+pinned: false
+types: ["solution"]
+areas: ["repo", "env", "Lotus Miner"]
+---
+
+
+# Problem
```

Comment:
```suggestion
## Problem
```

The `title: "StorageMiner repo directory does not exist"` acts as the `<h1>` header. Only ever have ` <h1>` header on a page. This is for accessibility stuff.